aur_install.go

```diff
@@ -27,6 +27,8 @@ type (
 		exeCmd           exe.ICmdBuilder
 		vcsStore         vcs.Store
 		targetMode       parser.TargetMode
+		reBuild          string
```

Comment:
Hey, I'll test it out this week,  

- I'd turn this into a `type rebuildMode string` and declare some constants to avoid dynamic behavior (user's putting the wrong value in config and thinking it's configure)

nit: Don't really care but https://www.merriam-webster.com/dictionary/rebuild does not need to be cased as it's its own word

Reply:
I also had a similar idea, but abandoned it, at least at this stage. It doesn't solve anything, since configuration machinery doesn't have any config value validation mechanism(perhaps this deserves a separate issue).

But if you insist that it's good to have it with an eye on the future, I'll do so.

I named the `Installer`'s field `reBuild` so just to be in line with the `Configuration`'s field `ReBuild`. I can rename both if I touch `Configuration`.